packages/ast-utilities/src/javascript/addImportStatement/addImportStatement.ts

```diff
@@ -19,5 +19,19 @@ export default function addImportStatement(statement: string | string[]) {
         path.node.body.unshift(ast);
       });
     },
+    File(path: traverse.NodePath<t.Program>) {
```

Comment:
Could we add a test for this case?

Reply:
Good catch! I can't with my current test utilities because the `File` node is not part of the fixture (they start at `Program` because they are coming from strings not files). I tried to get around this without any luck.

I suggest we merge without a test for this and I'll come back to it. These utilities are already well tested given how experimental they are, and this particular logic exists to catch the rare edge case that they run this transform on an empty file.

What do you think?